aws/table_aws_cloudwatch_log_group.go

```diff
@@ -69,6 +69,20 @@ func tableAwsCloudwatchLogGroup(_ context.Context) *plugin.Table {
 				Description: "The number of bytes stored.",
 				Type:        proto.ColumnType_INT,
 			},
+			{
+				Name:        "data_protection",
+				Description: "Log group data protection policy information.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getCloudwatchLogGroupDataProtectionPolicy,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "data_protection_policy",
+				Description: "The data protection policy document for a log group.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getCloudwatchLogGroupDataProtectionPolicy,
+				Transform:   transform.FromField("PolicyDocument"),
```

Comment:
Do we need any additional transforms, since the protection policy comes in as a string?

Reply:
@cbruno10 I didn't use any transform function, and I was able to query the column correctly. So I don't think we need it.

Reply:
What transforms do we use for similar columns in other tables, if any? Also when querying the column, were you able to interact with the column as if it were JSONB, e.g., use Postgres JSONB functions, access specific elements with ->?

Reply:
Yes @cbruno10 I was able to query the column without any transform function 👍.